DQM/EcalCommon/python/CommonParams_cfi.py

```diff
@@ -4,8 +4,8 @@
     onlineMode = cms.untracked.bool(False),
     willConvertToEDM = cms.untracked.bool(True)
 )
-ecaldqmLaserWavelengths = 1, 2, 3
-ecaldqmLedWavelengths = 1, 2
-ecaldqmMGPAGains = 12
-ecaldqmMGPAGainsPN = 16
 
+ecaldqmLaserWavelengths = cms.untracked.vint32(1, 2, 3)
```

Comment:
A safer way for updating those vectors of int is:
```suggestion
ecaldqmLaserWavelengths = [1, 2, 3]
ecaldqmLedWavelengths = [1, 2]
ecaldqmMGPAGains = [12]
ecaldqmMGPAGainsPN = [16]
```

Reply:
Thanks

Reply:
Sorry, I realized now your suggestion is a bit weird, we need to set them as cms.untracked here so, for my understanding, what do the brackets do in the previous lines?

Reply:
Sorry, my fault!
I had the impression that you were updating already existing parameters, not defining some new ones in the PSet (I was looking a the updated code too quickly, my bad...).
Please revert the last commit, so that we can merge this as it was

Reply:
Done. Thanks!